traffic_ops/app/lib/API/Tenant.pm

```diff
@@ -117,14 +123,25 @@ sub update {
 	my $is_active = $params->{active};
 	
 	if ( !$params->{active} && $self->isRootTenant($id)) {
```

Comment:
and if you make root tenant uneditable, then just make parentId a required field...